mvc/views/view-components.rst

```diff
@@ -1,16 +1,25 @@
-View Components
-======================================================
+View Components in MVC Core
```

Comment:
The title should just be View Components. These are the MVC docs so no need to restate that.